showcase/src/styles/styles.scss

```diff
@@ -1,8 +1,9 @@
 @import "theme";
 @import "stark-styles.scss";
 
-@import "~basscss/css/basscss.css";
-@import "~prismjs/themes/prism-okaidia.css";
-
+@import "../app/app.component.scss";
 @import "../app/news/news-component/news.component";
 @import "../app/news/news-item-component/news-item.component";
+
+@import "~basscss/css/basscss.css";
+@import "~prismjs/themes/prism-okaidia.css";
```

Comment:
Why are these imports now the last ones? If there is an important reason to do so we can add a comment to explain why otherwise I suggest we keep them in the place they were before since they are supposed to have no conflicts with the app styles